### DISCUSSION TITLE
Syntax Highlighting in VS Code

### DISCUSSION THREAD

**Discussion by pietz** [jpsca/jinjax#84] [Show and tell]
I couldn't find a nice way to give me proper Syntax Highlighting, so I threw together a quick VS Code extension for that:

https://marketplace.visualstudio.com/items?itemName=pietz.jinjax-syntax

I've never build one before, so there might be bugs. I decided to highlight the component names in a different color to make the difference a bit clearer to the eye. You need something like "Better Jinja" installed because it relies on vanilla Jinja highlighting.

**Comment by jpsca** [comment]
OMG, Thank you

**Comment by jpsca** [comment]
How do you activate it? After install it I don't see anything new in the language mode list (I have the ["Better Jinja"](https://marketplace.visualstudio.com/items?itemName=samuelcolvin.jinjahtml) extension)

![jinja1](https://github.com/user-attachments/assets/0253a513-a24c-4ac1-a378-f0b9e42c5564)

![jinja2](https://github.com/user-attachments/assets/d7163652-fec5-4c74-9f04-a77af7db9ae7)

**Reply by CiberNin** [reply]
I just installed it and it worked out of the box with better jinja and the Jinja-HTML language mode.

**Reply by CiberNin** [reply]
Note I did recently find that some extensions for Django etc may break the highlighting. You can use vscode developer inspect tokens and scopes to debug.

IMO this should be linked on the readme.
